src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-message.service.ts

```diff
@@ -146,7 +146,7 @@ export class TaskMessageService {
     'rbd/trash/restore': new TaskMessage(
       new TaskMessageOperation('Restoring', 'restore', 'Restored'),
       (metadata) =>
-        `image '${metadata.pool_name}/${metadata.image_name}@${metadata.image_id}' \
+        `image '${metadata.pool_name}@${metadata.image_id}' \
```

Comment:
Could you please add tests for the RBD trash notifications? As they don't exist yet.

Reply:
Done.